src/components/strategies/create/overlapping/CreateOverlappingStrategy.tsx

```diff
@@ -121,10 +130,14 @@ export const CreateOverlappingStrategy: FC<OverlappingStrategyProps> = (
   useEffect(() => {
     if (!quote || !base || marketPrice <= 0) return;
     if (!order0.min && !order1.max) {
-      const _marketPrice = new SafeDecimal(marketPrice);
-      const min = _marketPrice.times(0.999).toFixed(quote.decimals);
-      const max = _marketPrice.times(1.001).toFixed(quote.decimals);
-      if (isValidRange(min, max)) setOverlappingParams(min, max);
+      const { min, max } = getInitialPrice(new SafeDecimal(marketPrice), quote);
```

Comment:
I would move `new SafeDecimal(marketPrice)` into `getInitialPrice` and accept `number | string` as marketPrice argument type.